scripts/systemd/zeppelin.systemd

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Apache Zeppelin daemon
+After=syslog.target network.target
+
+[Service]
+Type=oneshot
+ExecStart=/opt/zeppelin-0.8.1-bin-all/bin/zeppelin-daemon.sh start
```

Comment:
Hi,  @monsieurp  The path may not be `/opt/zeppelin-0.8.1-bin-all/`?

Reply:
Hi @liuxunorg! Thanks for reviewing.

You are absolutely right: the directory in which Zeppelin files are to be installed can sit anywhere on the filesystem. I tend to install Zeppelin in `/opt` but that's just my personal preference. Of course, someone else may install Zeppelin in a different place and therefore, the path may not be `/opt/...` in this case. What do you suggest then? Because a path has to be specified in the systemd unit file.

Reply:
I have made this assumption very clear in the `README.md` file. Please have a look.

Reply:
Perhaps you can add a script for `add_daemon.sh` to `zeppelin/bin/`.
Use this script to generate the correct path to the `zeppelin.service` file in the `/etc/systemd/system` directory.

But you need to pay attention to the following points:
1. You need to determine whether the execution user is `root` in `add_daemon.sh`
2. You need to determine the version of the operating system in `add_daemon.sh`, because the path of the `systemd` folder of different operating systems may be different.
3. You need to determine the version of the operating system in `add_daemon.sh`, because the `systemctl` commands may be different for different operating systems.